packages/pods-core/src/builtin/AirtablePod.ts

```diff
@@ -390,7 +398,7 @@ export class AirtableUtils {
       records.map(async (ent) => {
         const airtableId = ent.id;
         const dendronId = ent.fields["DendronId"] as string;
-        const note = engine.notes[dendronId];
+        const note = engine.notes[dendronId] as NotePropsWithOptionalCustom;
```

Comment:
Is this actually necessary? what if we do something akin to `isStringNullOrEmpty()` on the NoteAirtableId below.  Can we avoid a cast and a new type definition?

Reply:
our actual `NoteProps` definition is wrong. `custom` is optional and can be undefined. we actually need to refactor the definition (or the code) but that was a much bigger change, hence this type as a stopgap.

we'll fix this as part of january core feature maturity milestones